### PR TITLE
Initial setup for wasm-lz4 project into Lichtblick

### PR DESCRIPTION
**Pull request by laisspportugal**
Description:

Initial setup for wasm-lz4 project into Lichtblick organization. It includes:

- Renaming

Checks performed:

- [x] Unit tests 
- [x] Verify build

### CODE REVIEW

File: package.json
Comment (aneuwald-ctw) [inline package.json:1]: I think we should remove this line (7) -> "repository": "cruise-automation/wasm-lz4".

Maybe better to use the same of others:

  "repository": {
    "type": "git",
    "url": "https://github.com/lichtblick-suite/wasm-lz4"
  },
  
 What do you think?

**Review by aneuwald-ctw — CHANGES_REQUESTED** [review]
Just update the repository information

File: package.json
Comment (laisspportugal) [reply package.json:1]: Fixed

**Review by aneuwald-ctw — APPROVED** [review]
lgtm :)

**Pull request merged**